idd/Energy+.idd.in

```diff
@@ -56508,7 +56493,7 @@ Coil:Cooling:WaterToAirHeatPump:ParameterEstimation,
         \memo evaporation from wet coil when compressor cycles off with continuous fan operation.
         \memo Parameter estimation model is a deterministic model that requires a consistent set of
         \memo parameters to describe the operating conditions of the heat pump components.
-        \min-fields 18
+        \min-fields 22
```

Comment:
This needs to extend all the way to the end of the object now, needs to include the last defaulted field.